[Backport] [7.80.x] Release integrations (#23708)
DataDog/integrations-core: kafka_actions/CHANGELOG.md

<!-- towncrier release notes start -->

## 2.7.0 / 2026-05-14

***Added***:

> * Support skipping a fixed number of bytes before deserialization via `value_skip_bytes` and `key_skip_bytes` for stripping producer-side prefixes. ([#23556](https://github.com/DataDog/integrations-core/pull/23556))

Reviewer: ```suggestion
* Support skipping a fixed number of bytes before deserialization through `value_skip_bytes` and `key_skip_bytes` for stripping producer-side prefixes. ([#23556](https://github.com/DataDog/integrations-core/pull/23556))
```

***Fixed***:

* Remove unused MessageFilter class ([#23430](https://github.com/DataDog/integrations-core/pull/23430))

> * Preload protobuf well-known types (e.g. `google/protobuf/timestamp.proto`) into the descriptor pool so schemas that reference them deserialize correctly. ([#23431](https://github.com/DataDog/integrations-core/pull/23431))

Reviewer: ```suggestion
* Preload protobuf well-known types (such as `google/protobuf/timestamp.proto`) into the descriptor pool so schemas that reference them deserialize correctly. ([#23431](https://github.com/DataDog/integrations-core/pull/23431))
```
